src/engine/qcommon/qcommon.h

```diff
@@ -285,7 +285,7 @@ You or the server may be running older versions of the game."
 
 #define PROTOCOL_VERSION       86
 
-#define URI_SCHEME             GAMENAME_STRING "://"
+#define URI_SCHEME             va("%s://", current_game->gamename_string)
```

Comment:
I agree, that's a protocol thing, so that's related to the engine. Perhaps unv:// is not enough reusable for a generic Dæmon thing, any suggest?